### ISSUE TITLE
Create textarea for konva component

### ISSUE THREAD

**Issue by brauliodiez** [Lemoncode/quickmock#26]
** Pending on trasnformer implementation to be merged **

Create a text area  for konva component

<img width="400" alt="image" src="https://github.com/Lemoncode/quickmock/assets/1457912/4794d28e-1d2f-4fd1-8fc5-f0f474be6ff6">

We have got the following text area SVG component:

https://github.com/brauliodiez/konba-playground/blob/main/public/widgets/textarea.svg?short_path=308a0c5

Implement a React.konba component that takes into consideration resizing, take a look to the combobox konba implementation on Quick Mock

https://github.com/Lemoncode/quickmock/blob/main/src/common/components/front-components/combobox-shape.tsx

Add a solid white background to the input inner content.

Right now we will support dynamic resizing, but we won't bother about adding a text property.

A possible starting point:

```tsx
import React, { forwardRef } from 'react';
import { Group, Rect, Text } from 'react-konva';
import { ShapeConfig } from 'konva/lib/Shape';

interface TextareaShapeProps extends ShapeConfig {
  id: string;
  x: number;
  y: number;
  width: number;
  height: number;
  onSelected: (id: string) => void;
}

export const TextareaShape = forwardRef<any, TextareaShapeProps>(
  ({ x, y, width, height, id, onSelected, ...shapeProps }, ref) => {
    const handleClick = () => {
      onSelected(id);
    };

    return (
      <Group x={x} y={y} ref={ref} {...shapeProps} onClick={handleClick}>
        {/* Rectángulo del textarea */}
        <Rect
          x={0}
          y={0}
          width={width}
          height={height}
          cornerRadius={10}
          stroke="black"
          strokeWidth={2}
          fill="none"
        />
        
        {/* Texto simulado dentro del textarea */}
        <Text
          x={10}
          y={30}
          text="Your text here..."
          fontFamily="Comic Sans MS, cursive"
          fontSize={20}
          fill="black"
        />
      </Group>
    );
  }
);

export default TextareaShape;
```

About styling we will create a separate task to extract common styles for all components (e.g. font used)
